etcdserver/raft.go

```diff
@@ -156,6 +156,7 @@ func (r *raftNode) start(s *EtcdServer) {
 					if lead := atomic.LoadUint64(&r.lead); rd.SoftState.Lead != raft.None && lead != rd.SoftState.Lead {
 						r.mu.Lock()
 						r.lt = time.Now()
+						leaderChanges.Inc()
```

Comment:
I'd keep this outside the mutex, it doesn't need to be protected and smaller locks are better.